lib/diagnostic.js

```diff
@@ -47,7 +50,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.bind(alderaan.name);
```

Comment:
`.bind` expects an object as an argument to that it can (temporarily) make it the new context of the given function; `alderaan.name` is actually a String. 

`.bind` also does not invoke the bound function; you need to do that yourself.
Alternatively, you could use `.call` or `.apply` to bind **AND** invoke.